src/protocol/PasLS.Symbols.pas

```diff
@@ -349,8 +349,8 @@ constructor TSymbolTableEntry.Create(_Code: TCodeBuffer);
 { TSymbolExtractor }
 
 procedure TSymbolExtractor.PrintNodeDebug(Node: TCodeTreeNode; Deep: boolean);
-var
-  Child: TCodeTreeNode;
+{var
```

Comment:
if it's unused probably just remove this line?

Reply:
I didn't because there was some code in comment that seemed to use the child ?